src/textures.c

```diff
@@ -221,12 +222,12 @@ int texLookupInternalTexId(const char *name)
     return result;
 }
 
-static int texSizeValidate(int width, int height, short psm)
+static int texSizeValidate(int width, int height, u8 psm)
 {
     if (width > 1024 || height > 1024)
         return -1;
 
-    if (gsKit_texture_size(width, height, psm) > maxSize)
+    if (gsKit_texture_size(width, height, (short)psm) > maxSize)
```

Comment:
why set it to (short) ? It is already limited in declaration.

Reply:
Variable preservation when the function was written i will roll back and add int instead but this was done on purpose.

Reply:
Solved.